dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -204,12 +225,22 @@ def profile(self):
 
         :return:
         """
+        num_most_common_words = self._num_most_common_words
+        if num_most_common_words is None:
+            num_most_common_words = len(self.word_count.keys())
+
+        num_most_common_chars = self._num_most_common_chars
+        if num_most_common_chars is None:
+            num_most_common_chars = len(self.vocab_count.keys())
```

Comment:
I believe `Counter` accepts `None`, hence no need to do this.

Reply:
good point. I removed this part.